### PR TITLE
Basic data error handling

### PR DESCRIPTION
**Pull request by seifriedc**
For issue #31 - should catch bad response and no records errors. Each error box has a retry button which closes the box when clicked.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -437,7 +458,6 @@
         console.log('All data loaded');
```
Comment (vonbearshark) [inline main.js]: We can take this out

File: main.js [hunk 2]
```diff
@@ -437,7 +458,6 @@
...
     }).catch((err) => {
         console.log('final error catch data', err);
```
Comment (vonbearshark) [inline main.js]: We should have a display here that retriggers the `Promise.all`

**Review by vonbearshark — COMMENTED** [review]
Nice work! Just one more display for `Promise.all`

File: main.js (hunk 2 above)
Comment (seifriedc) [reply main.js]: How would you do this, just make Promise a variable and call it? Sorry lol new to JS

Comment (vonbearshark) [reply main.js]: The simplest way might be to wrap this chunk that calls `Promise.all` in a function--something like `fetchAllData`. Then on the "retry" button click handler we could just call `fetchAllData` again.

**Review by vonbearshark — APPROVED** [review]
Looks awesome!

**Pull request merged**